scripts/import_wiseconnect.py

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env python3
+
+"""
+Copyright (c) 2024 Silicon Laboratories Inc.
```

Comment:
2025 perhaps?

Reply:
In fact I wrote this script in November 2024:
```
From e7c21e3b0fdd3153b32ad2a4d9d15353f97b6aca Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?J=C3=A9r=C3=B4me=20Pouiller?= <jerome.pouiller@silabs.com>
Date: Fri, 22 Nov 2024 14:22:15 +0100
Subject: [PATCH] wiseconnect: Introduce import script
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
```